tests/test_app.py

```diff
@@ -44,20 +45,12 @@
 AUTH_URL = CONFIG.auth_url
 
 
-def asyncgiven(**kwargs):
-    """alternative to hypothesis.given decorator for async"""
-
-    def real_decorator(fn):
-        @given(**kwargs)
-        def aio_wrapper(*args, **kwargs):
-            loop = asyncio.new_event_loop()
-            asyncio.set_event_loop(loop)
-            future = asyncio.wait_for(fn(*args, **kwargs), timeout=60)
-            loop.run_until_complete(future)
-
-        return aio_wrapper
-
-    return real_decorator
+@pytest.fixture(scope="function")
```

Comment:
Something like this should supercede the FileUtil context thing eventually, IMO.

This monkeypatches the config object to use a `tmp_path` for the DTS dir for tests. So we don't nuke configured directories in the user's filesystem.